src/spring/HISTORY.md

```diff
@@ -1,5 +1,10 @@
 Release History
 ===============
+1.9.1
+---
+* Support subPath for bring your own persistent storage feature.
+* Add new parameter `--enable-sub-path` into `az spring append-persistent-storage` to enable subPath feature.
```

Comment:
If you want to release a new extension version for this PR, please also upgrade the version defined in the `setup.py`